api_functional_v4_tests.go

```diff
@@ -27,22 +29,34 @@ import (
 	"net/http"
 	"net/url"
 	"os"
+	"runtime"
 	"strconv"
 	"strings"
-	"testing"
 	"time"
 
+	minio "github.com/minio/minio-go"
+	log "github.com/sirupsen/logrus"
+
 	"github.com/minio/minio-go/pkg/encrypt"
 	"github.com/minio/minio-go/pkg/policy"
 )
 
+const minPartSize = 1024 * 1024 * 64
 const letterBytes = "abcdefghijklmnopqrstuvwxyz01234569"
 const (
 	letterIdxBits = 6                    // 6 bits to represent a letter index
 	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
 	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
 )
 
+func getFilePath(filename string) string {
+	var dir = os.Getenv("DATA_DIR")
```

Comment:
Use the named returns here. 

```
func getFilePath(filename string) (filePath string) {
            dataDir := os.Getenv("DATA_DIR")
            if dataDir == "" {
                     dataDir = "/mint/data"
            }
            return dataDir + "/" + filename
}
```

Also make getDataDir() another function to return the configured DATA_DIR only and then use it with `getFilePath()`